app/client/components/FormRenderer.ts

```diff
@@ -423,13 +423,13 @@ class ChoiceRenderer extends BaseFieldRenderer  {
   }
 
   public fieldDomAttributes() {
-    if (this._format === 'select') {
-      return {};
+    if (this._format === 'radio') {
```

Comment:
Can you convert it to switch with a default clause that will throw the error? Right now I don't see a difference, it isn't more explicit to me.